app/bundles/CategoryBundle/Translations/en_US/messages.ini

```diff
@@ -1,5 +1,5 @@
 mautic.category.createnew="Create new category..."
-mautic.category.form.alias.help="Letters and numbers (hyphens allowed) used for URL generation of this category. A unique alias based on the title will be autogenerated if left empty."
+mautic.category.form.alias.help="Letters and numbers (hyphens allowed) are used for URL generation of this category. A unique alias based on the title will be autogenerated if left empty."
```

Comment:
This one won't make sense in the context where it's used

Reply:
reverted